site/content/docs/user/nvidia-dranet.md

```diff
@@ -98,9 +98,11 @@ kubectl apply -f https://raw.githubusercontent.com/google/dranet/refs/heads/main
 In order to install the NVIDIA DRA Drivers you will need to clone the [NVIDIA DRA](https://github.com/NVIDIA/k8s-dra-driver-gpu) repo. Ensure you have [helm](https://helm.sh/docs/intro/install/) installed.
 
 ```
-helm upgrade -i --create-namespace --namespace nvidia-dra-driver-gpu nvidia-dra-driver-gpu ./k8s-dra-driver-gpu/deployments/helm/nvidia-dra-driver-gpu --set gpuResourcesEnabledOverride=true --set nvidiaDriverRoot=/home/kubernetes/bin/nvidia --wait 
+helm upgrade -i --create-namespace --namespace nvidia-dra-driver-gpu nvidia-dra-driver-gpu ./k8s-dra-driver-gpu/deployments/helm/nvidia-dra-driver-gpu --set gpuResourcesEnabledOverride=true --values https://raw.githubusercontent.com/google/dranet/refs/heads/main/values.yaml --wait 
 ```
 
+The values.yaml adds some additional tolerations and removes some priorities that need to be done in order to work nicely with GKE.
```

Comment:
If we are targeting GKE (to begin with, since that is where we can test things), is it reasonable to limit our values.yaml file to just the things we have to override and leave the omit the rest of the things completely?

For example, based on https://cloud.google.com/kubernetes-engine/docs/how-to/set-up-dra#driver-gpu, it seems like we only need a few fields.

Reply:
I feel like this way is better since in this case we are going through a specific example and there is already a lot of flags in that example given. We might be able to make the values.yaml a bit smaller but I think we’d need to import both the base values and these. I think a shorter command line is generally preferable imo.

Reply:
Yeah I can understand that. My reasoning is not so that we have a small file, but more so that we derive any defaults that are available in the upstream helm chart and don't override anything which we don't have to (whose implications we may not fully understand). 

I think the default derivation should work, but I won't want you spending significant time with this. I'm totally fine with you time boxing this and we can merge the PR as it is.

Reply:
I played around with it a touch, this I think is the smallest I can get, let me know what you think!

Reply:
Thank you :)